packages/injective-cosmwasm/src/types.rs

```diff
@@ -28,6 +30,16 @@ impl MarketId {
         self.0.as_str()
     }
 
+    pub fn validate(self, querier: InjectiveQuerier, is_spot: bool) -> StdResult<Self> {
```

Comment:
what do you think about passing some enum here instead of bool? it would make function calls easier to understand. e.g. 
```
    pub enum MarketType {
        Spot,
        Derivative
    }
```

also, why not `querier: &InjectiveQuerier`?